README.md

```diff
@@ -226,6 +224,7 @@ Create a new instance of `merry`. Takes optional opts:
   logging
 - __opts.logStream:__ defaults to `process.stdout`. Set the output writable stream to
   write logs to
+- __opts.logName:__ defaults to `merry`. Sets the name of the logger you're starting
```

Comment:
I don't think this option is ideal; I find it slightly confusing to name parts of the app, but not the app as a whole. If we're allowing people to name things (and I'm not saying we should), it should probably be a string as the first parameter in the constructor, ("opts" as second), or a "name" key on the "opts" object. In both cases it should still be optional